src/Generator/Property/PropertySetter.cs

```diff
@@ -5,33 +5,16 @@ namespace Microsoft.DigitalWorkplace.DigitalTwins.Models.Generator;
 
 internal class PropertySetter : PropertyAccessor
 {
-    internal AccessModifier? AccessModifier { get; set; }
 
-    internal bool IsInit { get; set; }
+    internal bool IsInit { get; }
 
     internal PropertySetter(ModelGeneratorOptions options) : base(options)
     {
     }
 
     internal void WriteTo(StreamWriter writer)
     {
-        if (AccessModifier != null)
-        {
-            writer.Write(AccessModifier?.Serialize());
-            writer.Write(" ");
-        }
-
         writer.Write(IsInit ? "init" : "set");
```

Comment:
```suggestion
        writer.Write("set; ");
```
... and remove the next line.